src/md/enc/metamodelrw.cpp

```diff
@@ -8780,7 +8780,7 @@ bool FilterTable::IsTokenMarked(
         //
         return true;
     }
-    return ( (*Get(rid)) & bitMarked ? true : false);
+    return ( ((*Get(rid)) & bitMarked) ? true : false);
```

Comment:
@Dmitry-Me I guess that it due to the area you are touching. The previous PR was in PAL, which is Unix / CoreCLR specific. This one touches code that I think is common with full desktop CLR.
@richlander, am I right?